README.md

```diff
@@ -1,9 +1,81 @@
 # Wiki Loves Monuments
 
-This repo is a collection of code supporting the Wiki Loves Monuments competition on the German Wikipedia.
+This repo is a collection of code enabling easier image uploads on the German Wikipedia for Wiki Loves Monuments competition.
 
-At the moment it consists of bots that analyze and prepare the monument pages and a PHP script that forwards from Wikipedia links to the Commons Uploader, adding URL parameters.
+The code consists of two parts:
+1. Bots that analyze the monument pages and insert images from Commons in the monument lists. See the [bots README](update-bot/README.md) for detailed documentation on the bots.
+2. A PHP script that forwards from Wikipedia links to the Commons Uploader, adding URL parameters. See the [forward script README](forward_script/README.md) for detailed documentation on the forward script.
 
-See the [bots README](update-bot/README.md) for detailed documentation on the bots.
+## Deployment on Tool Labs
+The tool name is `wlm-de-utils`. All following instructions must be carried out with the tool account, *not* your personal Tool Labs account. To switch to the tool account, run the following command after logging in with your personal Tool Labs account:
 
-See the [forward script README](forward_script/README.md) for detailed documentation on the forward script.
+    become wlm-de-utils
+
+Generally, the [`WikiLovesMonuments`][wlmrepo] repository is checked out in the tool home folder.
```

Comment:
I know I am about to start a flame war here but: I'd suggest "directory" over "folder"